src/main/scala/beam/agentsim/agents/ridehail/RideHailManager.scala

```diff
@@ -306,8 +293,16 @@ class RideHailManager(
   // Are we in the middle of processing a batch?
   var currentlyProcessingTimeoutTrigger: Option[TriggerWithId] = None
 
+  private val numHouseholdVehicles = scenario.getHouseholds.getHouseholds
+    .values()
+    .asScala
+    .map(_.getVehicleIds.size())
```

Comment:
This could be done with a `reduce` or `fold` instead of a `map/sum` - which should be more performant I believe.